app/js/features/metadata/reducer.js

```diff
@@ -0,0 +1,29 @@
+import {
+  LOAD_OFFERS_SUCCEEDED, LOAD_SELLER_SUCCEEDED
+} from './constants';
+
+const DEFAULT_STATE = {
+  sellers: {},
+  offers: {}
+};
+
+function reducer(state = DEFAULT_STATE, action) {
+  switch (action.type) {
+    case LOAD_SELLER_SUCCEEDED:
+      return {
+        ...state, ...{
```

Comment:
I don't think you need to spread the `sellers` object, it will eval to the same as 
```
{...state, sellers: { ...state.sellers, [action.address]: action.seller} }
```
but with an extra operation

Reply:
:+1: updated